stats/worker.go

```diff
@@ -272,10 +272,28 @@ func (w *worker) tryRegisterView(v *View) error {
 
 	w.viewsByName[v.Name()] = v
 	w.views[v] = true
+
+	// If any exporter implements ViewRegistrar, broadcast to it
+	go w.broadcastViewToExporters(v)
```

Comment:
Am not sure, I figured since it is a broadcast to exporters that might or might not have a RegisterView method, that it shouldn't block. If you'd like though, I can make it block.